### PR TITLE
New shoe sprites

### PR DESCRIPTION
**Pull request by alsoandanswer**
what it says.

replaced leather shoes with brown oxford shoes (basically the same, but sounds fancier)

adds new workboot types (and toeless variant)

new toeless winterboots

new oxford shoes selection

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/modules/clothing/shoes/colour.dm [hunk 1]
```diff
@@ -104,6 +98,33 @@
 	icon_state = "orange"
 	var/obj/item/handcuffs/chained = null
 
+/obj/item/clothing/shoes/orange/proc/attach_cuffs(var/obj/item/handcuffs/cuffs, mob/user as mob)
```
Comment (NonQueueingMatt) [inline code/modules/clothing/shoes/colour.dm]:
```suggestion
/obj/item/clothing/shoes/orange/proc/attach_cuffs(var/obj/item/handcuffs/cuffs, mob/user)
```

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

**Comment by KasparoVy** [comment]
Swanky. We high class spacefolk now

**Pull request merged**